modules/memegenBidAdapter.js

```diff
@@ -0,0 +1,127 @@
+var CONSTANTS = require('src/constants.json');
+var utils = require('src/utils.js');
+var bidfactory = require('src/bidfactory.js');
+var bidmanager = require('src/bidmanager.js');
+var adloader = require('src/adloader');
+var adaptermanager = require('src/adaptermanager');
+import {config} from 'src/config';
+
+var bidderName = 'memegen';
+/**
+ * Adapter for requesting bids from Meme Global Media Group
+ * OpenRTB compatible
+ */
+var MemeGlobalAdapter = function MemeGlobalAdapter() {
+  var openRtbAdapterHost = 'ssp.velemh.com';
+  var timeout;
+  function _callBids(params) {
+    timeout = params.timeout;
+    var bids = params.bids;
+
+    if (!bids) return;
+
+    for (var i = 0; i < bids.length; i++) {
+      _requestBid(bids[i]);
+    }
+  }
+
+  function _requestBid(bidReq) {
+    timeout = timeout || config.getConfig('bidderTimeout');
+    // build bid request object
+    var domain = window.location.host;
+    var page = window.location.host + window.location.pathname + location.search + location.hash;
```

Comment:
this will have the domain issue we're seeing now, right? 
lets see how we can tackle it (in the next PRs)